pom.xml

```diff
@@ -214,6 +216,7 @@
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-gpg-plugin</artifactId>
+            <version>1.6</version>
```

Comment:
This is an unrelated change but it could be included since improves the readability and avoid problems.